### ISSUE TITLE
KeyError for token when setting authorization header

### ISSUE THREAD

**Issue by foospidy** [foospidy/pysigsci#21]
Reported from user using the sigsci_site_manager (https://github.com/turnerlabs/sigsci_site_manager)

```
Traceback (most recent call last):
  File "C:\Program Files\Python37\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Program Files\Python37\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "H:\sigsci\venv\Scripts\sigsci_site_manager.exe\__main__.py", line 9, in <module>
  File "h:\sigsci\venv\lib\site-packages\sigsci_site_manager\site_manager.py", line 291, in main
    args.func(args)
  File "h:\sigsci\venv\lib\site-packages\sigsci_site_manager\site_manager.py", line 22, in do_list
    resp = api.get_corp_sites()
  File "h:\sigsci\venv\lib\site-packages\pysigsci\sigsciapi\sigsciapi.py", line 266, in get_corp_sites
    endpoint="{}/{}/sites".format(self.ep_corps, self.corp))
  File "h:\sigsci\venv\lib\site-packages\pysigsci\sigsciapi\sigsciapi.py", line 48, in _make_request
    headers["Authorization"] = "Bearer {}".format(self.bearer_token['token'])
KeyError: 'token'
```

**Comment by ebpmp** [comment]
I put a simple patch in place for this:
https://github.com/foospidy/pysigsci/pull/23

**Comment by foospidy** [comment]
This should be fixed. Published new release 3.17.0 https://pypi.org/project/pysigsci/
